src/content/de/updates/2018/01/crux.md

```diff
@@ -0,0 +1,34 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Announcing the release of a new country dimension in the Chrome User Experience Report.
```

Comment:
Should we translate the description?

Reply:
Quite possibly... I can put a special handler in for it.

Reply:
Nice to have, not a priority.